imageroot/actions/get-configuration/20read

```diff
@@ -20,6 +39,7 @@ config = {
     'reports_international_prefix': os.getenv('REPORTS_INTERNATIONAL_PREFIX', ''),
     'nethvoice_adm_username': os.getenv('NETHVOICE_USER_PORTAL_USERNAME', ''),
     'nethvoice_adm_password': agent.read_envfile("passwords.env").get('NETHVOICE_USER_PORTAL_PASSWORD',""),
+    "proxy_installed": proxy_installed,
```

Comment:
The action `get-configuration` must return as output the same the same input of the `configure-module` action.
We need to find another way to get the proxy status.
Can be another action or extend the `configure-module` to accept the proxy moduleid/uuid

Reply:
it should be good